lib/ace/ext/papeeria/papeeria_latex_highlight_rules.coffee

```diff
@@ -7,26 +7,39 @@ define((require, exports, module) ->
 
   START_STATE = "start"
 
+  LIST_ITEMIZE_STATE = "list.itemize"
+  LIST_ITEMIZE_REGEX = "itemize"
+
+  LIST_ENUMERATE_STATE = "list.enumerate"
+  LIST_ENUMERATE_REGEX = "enumerate"
+
+  EQUATION_REGULAR_STATE = "equation.regular"
+  EQUATION_REGULAR_REGEX = "equation"
+
+  EQUATION_ASTERISK_STATE = "equation.asterisk"
+  EQUATION_ASTERISK_REGEX = "equation\\*"
+
+  MATH_MULTILINE_STATE = "math.multiline"
+  MATH_MULTILINE_OPENING_REGEX = MATH_MULTILINE_CLOSING_REGEX = "\\$\\$"
+
+  MATH_INLINE_STATE = "math.inline"
+  MATH_INLINE_OPENING_REGEX = MATH_INLINE_CLOSING_REGEX = "\\$"
+
+  MATH_ALTERNATIVE_STATE = "math.alternative"
+  MATH_ALTERNATIVE_OPENING_REGEX = "\\\\\\["
+  MATH_ALTERNATIVE_CLOSING_REGEX = "\\\\\\]"
+
   LIST_STATE = "list"
-  LIST_REGEX = "itemize|enumerate"
   LIST_TOKENTYPE = "list"
-
   EQUATION_STATE = "equation"
-  EQUATION_REGEX = "equation|equation\\*"
   EQUATION_TOKENTYPE = "equation"
 
-  MATH_STATE = "math"
-  MATH_CLOSING_REGEX = "\\${1,2}"
-
-  MATH_LATEX_STATE = "math_latex"
-  MATH_LATEX_CLOSING_REGEX = "\\\\\\]"
-
-  exports.EQUATION_STATE = EQUATION_STATE
   exports.LIST_STATE = LIST_STATE
-  exports.EQUATION_TOKENTYPE = EQUATION_TOKENTYPE
   exports.LIST_TOKENTYPE = LIST_TOKENTYPE
+  exports.EQUATION_STATE = EQUATION_STATE
+  exports.EQUATION_TOKENTYPE = EQUATION_TOKENTYPE
   PapeeriaLatexHighlightRules = ->
-    ###
+    ###*
```

Comment:
The reason the asterisk is there is because multiline comments delimited with `###` in CoffeeScript are translated into multiline comments in JavaScript delimited with `/*` and `*/`. And the convention for documention multiline comments is to begin not with `/*` but with `/**`. So writing `###*` translates to `/**` in JavaScript. But this really is a minor issue and it does look like a typo, so I'll just delete it, I guess.